website/content/docs/k8s/installation/deployment-configurations/clients-outside-kubernetes.mdx

```diff
@@ -66,91 +68,100 @@ clusters](https://cloud.google.com/kubernetes-engine/docs/concepts/alias-ips).
 option to expose the clients and server ports on the host IP instead.
 
-  1. Install the [official Helm chart](/docs/k8s/helm) with the following values:
-  ```yaml
-  client:
-    exposeGossipPorts: true # exposes client gossip ports as hostPorts
-  server:
-    exposeGossipAndRPCPorts: true # exposes the server gossip and RPC ports as hostPorts
-    ports:
-      # Configures the server gossip port
-      serflan:
-        # Note that this needs to be different than 8301, to avoid conflicting with the client gossip hostPort
-        port: 9301
-  ```
-   This will expose the client gossip ports, the server gossip ports and the server RPC port at `hostIP:hostPort`. Note that `hostIP` is the **internal** IP of the VM that the client/server pods are deployed on.
-
-  1. Make sure the IPs of the Kubernetes nodes are routable from the VM and
-  that the VM can access ports 8301 and 9301 (for gossip) and port 8300 (for
-  server RPC) on those node IPs.
-
-  1. Make sure the client and server pods running in Kubernetes can route to
-  the VM's advertise IP on its gossip port (default 8301).
-
-  3. Make sure you have the `kubeconfig` file for the Kubernetes cluster in `$HOME/.kube/config` on the external VM.
-
-  4. On the external VM, run (note the addition of `host_network=true` in the retry-join argument):
-  ```bash
-  consul agent \
-    -advertise="$ADVERTISE_IP" \
-    -retry-join='provider=k8s host_network=true label_selector="app=consul,component=server"'
-    -bind=0.0.0.0 \
-    -hcl='leave_on_terminate = true' \
-    -hcl='ports { grpc = 8502 }' \
-    -config-dir=$CONFIG_DIR \
-    -datacenter=$DATACENTER \
-    -data-dir=$DATA_DIR \
-  ```
-  3. Check if the join was successful by running `consul members`. Sample output:
-  ```shell-session
-  / $ consul members
-  Node                                           Address           Status  Type    Build  Protocol  DC   Segment
-  consul-consul-server-0                         10.138.0.43:9301  alive    server  1.9.1  2         dc1  <all>
-  external-agent                                 10.138.0.38:8301  alive    client  1.9.0  2         dc1  <default>
-  gke-external-agent-default-pool-32d15192-grs4  10.138.0.43:8301  alive    client  1.9.1  2         dc1  <default>
-  gke-external-agent-default-pool-32d15192-otge  10.138.0.44:8301  alive    client  1.9.1  2         dc1  <default>
-  gke-external-agent-default-pool-32d15192-vo7k  10.138.0.42:8301  alive    client  1.9.1  2         dc1  <default>
-  ```
+1. Install the [official Helm chart](/docs/k8s/helm) with the following values:
+
+```yaml
+client:
+  exposeGossipPorts: true # exposes client gossip ports as hostPorts
+server:
+  exposeGossipAndRPCPorts: true # exposes the server gossip and RPC ports as hostPorts
+  ports:
+    # Configures the server gossip port
+    serflan:
+      # Note that this needs to be different than 8301, to avoid conflicting with the client gossip hostPort
+      port: 9301
+```
```

Comment:
````suggestion
    ```yaml
    client:
      exposeGossipPorts: true # exposes client gossip ports as hostPorts
    server:
      exposeGossipAndRPCPorts: true # exposes the server gossip and RPC ports as hostPorts
      ports:
        # Configures the server gossip port
        serflan:
          # Note that this needs to be different than 8301, to avoid conflicting with the client gossip hostPort
          port: 9301
    ```
````